src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -64,6 +64,7 @@ def load_arguments(self, _):
             c.argument('virtual_network_rules', nargs='+', validator=validate_virtual_network_rules, help='ACL\'s for virtual network')
             c.argument('enable_multiple_write_locations', arg_type=get_three_state_flag(), help="Enable Multiple Write Locations")
             c.argument('disable_key_based_metadata_write_access', arg_type=get_three_state_flag(), help="Disable write operations on metadata resources (databases, containers, throughput) via account keys")
+            c.argument('public_network_access', help="Enable or disable public network access to server.")
```

Comment:
Could you please use enable_xxx like sql? 
https://github.com/Azure/azure-cli/blob/298994660f0fde6863cb45a7c3142141ed10f923/src/azure-cli/azure/cli/command_modules/sql/_params.py#L1069-L1074